ui/tests/integration/components/sidebar/nav/cluster-test.js

```diff
@@ -84,31 +84,6 @@ module('Integration | Component | sidebar-nav-cluster', function (hooks) {
     });
   });
 
-  test('it should render badge for promotional links on community version', async function (assert) {
```

Comment:
Should we have a test that it renders the HVD `plus` badge?

Reply:
It still does, just farther down https://github.com/hashicorp/vault/blob/main/ui/app/services/permissions.js#L254